solve/문제 풀이/재귀/25501.py

```
@@ -0,0 +1,15 @@
+def palindrome(s, l, r):
```

Ruff (0.13.3)

Ambiguous variable name: `l`

(E741)